test/helpers/index.js

```diff
@@ -5,6 +5,7 @@ import {finished} from 'node:stream/promises';
 export {fromAnyIterable as readableStreamFrom} from '@sec-ant/readable-stream/ponyfill';
 
 export const createStream = streamDefinition => typeof streamDefinition === 'function'
+	// eslint-disable-next-line n/no-unsupported-features/node-builtins
 	? Duplex.from(streamDefinition)
```

Comment:
`Duplex.from()` is supported in Node 18 but is marked as experimental.
However, its implementation has barely changed in many years now. Also, this is only used in tests.